alembic/versions/6f96516c7a7b_initial.py

```diff
@@ -0,0 +1,90 @@
+"""initial
+
+Revision ID: 6f96516c7a7b
+Revises:
+Create Date: 2022-10-06 06:50:45.512958+00:00
+
+"""
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "6f96516c7a7b"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
```

Comment:
Do we want to drop these old tables from the database entirely?

Reply:
I don't think this migration is needed anymore. When create a new CM instance, I don't see a complaints table.

Reply:
And actually in my testing these seem to be causing the migrations to fail.

Reply:
I pushed a commit that dropped this migration, so I could get a bit further with testing.